src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/RecallAuditComponentTest.kt

```diff
@@ -147,7 +148,9 @@ class RecallAuditComponentTest : ComponentTestBase() {
     // assert that audit still works when sentencing info - which has nested values - is updated
     authenticatedClient.updateRecall(recallId, UpdateRecallRequest(sentenceLength = Api.SentenceLength(3, 0, 0), sentenceDate = today, licenceExpiryDate = today, indexOffence = "Offence 1", sentencingCourt = CourtId("ACCRYC"), sentenceExpiryDate = today))
```

Comment:
This and similar length lines could use wrapping/re-formatting.

Reply:
I actually like that they are a single line 😬